ta/mk/lib.mk

```diff
@@ -0,0 +1,77 @@
+# Get the dir of the ta-dev-kit, requires make version 3.81 or later
```

Comment:
Yes, i think best way is to get it towards to the android build way, library and ta use the same included   common base makefile, but use different target type.  And  i think at least this patch is in the right direction.